Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
 name = "salsa"
-version = "0.1.1"
+version = "0.2.0"
 authors = ["Niko Matsakis <niko@alum.mit.edu>"]
 edition = "2018"
 license = "Apache-2.0/MIT"
```

Comment:
I think `Apatch-2.0 OR MIT` is the correct syntax.

Reply:
# Multiple licenses can be separated with a `/`, although that usage
# is deprecated.  Instead, use a license expression with AND and OR
# operators to get more explicit semantics.

Reply:
Huh, really? I've been using this forever...

Reply:
> Multiple licenses can be separated with a `/`, although that usage is deprecated.  Instead, use a license expression with AND and OR operators to get more explicit semantics.

TIL, ok.

Reply:
Still feels ambiguous to me though -- is it "OR" from the pov of a consumer or a provider. =) Ah well.